project_and_code_guidelines.md

```diff
@@ -218,7 +215,7 @@ class MyClass {
 
 Braces around the statements are required unless the condition and the body fit on one line. 
 
-This is good: 
+If the condition and the body fit in one line shorter than the max line length, it is recommended __not__ to use braces like the example below.
```

Comment:
this line is a bit confusing, it could be made clearer, maybe:

`If the condition and the body fit on one line and that line is shorter than the max line length, then do __not__ use braces e.g.`